core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -562,10 +577,16 @@ $(document).ready(function () {
                         }
                     ],
                     "columnDefs": [
-                        { "visible": false, "targets": 15 },
-                        { "visible": false, "targets": 16 },
-                        { "visible": false, "targets": 17 },
-                        { "visible": false, "targets": 18 }
+                        // String with structures like : 'bytes / bytes', 'bytes / bytes'
+                        // they should be sorted as numerical-order instead of lexicographical-order.
+                        { "type": "size", "targets": 9 },
```

Comment:
nit: add comments about which columns are these targets

Reply:
@gengliangwang  done this.